services/nexus/nexus.service.js

```diff
@@ -1,124 +1,214 @@
 'use strict'
```

Comment:
I considered renaming the service file to include `version` as these badges are all about versions from Nexus, and there's potentially other badge info (downloads for example) that could be included from Nexus in the future. I also wondered if supporting Nexus 2.x & 3.x would require different service classes (and thus different files).

In the end though I decided to leave the file name as is since we can always change it later if/when those other things come